solutions/stubs/src/generateRequestObject/handlers/create.ts

```diff
@@ -87,6 +87,9 @@ export function createRequestObjectPost(fastify: FastifyInstance) {
     url.searchParams.append("response_type", "code");
     url.searchParams.append("redirect_uri", redirectUrl);
     url.searchParams.append("request", result.encryptedJar);
+    if (typeof result.jwtPayload["state"] === "string") {
```

Comment:
can we provide a section to provide a state value in the stub ui, if not supplied it generates a random state in the request object and it is included in the authorize url qsp